lib/dor/datastreams/workflow_ds.rb

```diff
@@ -24,7 +23,7 @@ def get_workflow(wf, repo = 'dor')
       Workflow::Document.new(xml.to_s)
     end
 
-    alias_method :[], :get_workflow
+    alias :[] :get_workflow
```

Comment:
That said, I wouldn't worry too much about it. I think we should try to remove the `[]` accessor entirely. It seems to be used in one test in `dor-services`, but i'll take a look in some other projects just to be sure.